### ISSUE TITLE
Naming conflicts after updating to 0.7

### ISSUE THREAD

**Issue by DeFUCC** [antfu/vite-plugin-components#25]
I'm using the plugin with `directoryAsNamespace: true,` option. So for example I have my component `app-header` stored at  `components/app/header.vue`. I worked fine until I've updated the version from 0.6.6 to 0.7.1. Now I have such errors in the console:

```
[vite-plugin-components] component "Card"(....../src/components/def/card.vue) has naming conflicts with other components, ignored.
[vite-plugin-components] component "List"(....../src/components/def/list.vue) has naming conflicts with other components, ignored.
[vite-plugin-components] component "Card"(......./src/components/link/card.vue) has naming conflicts with other components, ignored.
... and so on...
```
Looks like a bug. Or may be I need to make some changes to the config?

**Comment by antfu** [comment]
The only thing in 0.7.1 is the Vue 2 transformer and it should not affect the logic related to your problem. Can you share a minimal repro? Thanks,

**Comment by DeFUCC** [comment]
@antfu no problem! https://github.com/DeFUCC/vite-min here's a minimal repro repo ) Two components work fine until you bump `vite-plugin-components` from 0.6.12 to 0.7.1

**Comment by antfu** [comment]
Should be fixed in 0.7.2. Thanks
